tests/RobotFramework/tests/cumulocity/supported_operations/mapper-publishing-agent-supported-ops.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Resource    ../../../../resources/common.resource
+Library    Cumulocity
+Library    ThinEdgeIO
+
+Test Tags    theme:c8y    theme:operation    theme:tedge-agent
+Test Setup    Custom Setup
+Test Teardown    Get Logs
+
+*** Test Cases ***
+Publish the tedge agent supported operations delete restart
+    # wait till list request is pushed out
+    Should Have MQTT Messages    tedge/commands/req/software/list
```

Comment:
Why do you need this step?

Reply:
Refactored. I was thinking just to make sure the `list` command is issued upon starting `tedge-mapper-c8y`. Now its moved to the second test and not required here anymore. This one tests the creation of operation files and publishing of operations.